android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -229,7 +229,7 @@ public void initKMKeyboard(final Context context) {
 
     getSettings().setCacheMode(WebSettings.LOAD_NO_CACHE);
     getSettings().setSupportZoom(false);
-
+    getSettings().setMinimumFontSize(4);
```

Comment:
We could go as low as `1` (as in, `1px`), but that might become too unreadable.

Reply:
1. Can we add the units (px?) in a comment?
2. How come iOS doesn't need this change?

Reply:
Re point 2... good question.  I guess it's a matter of the decisions by the devs of each browser?  I documented my investigation and its results on the base issue.

Point 1 - sure, why not?